requirements-dev.txt

```diff
@@ -0,0 +1,5 @@
+# Linters
+pylama>=7.1.0
+pylama-pylint>=2.2.1
+isort>=4.2.5
```

Comment:
You should not force other developers to use the same tools that you are using, right ? This is not a requirement, maybe a suggestion on HACKING.md. Unless that this is used as a syntax test check, is that the case ? I'm not sure.